native/src/main/java/io/ballerina/stdlib/grpc/builder/syntaxtree/utils/CommonUtils.java

```diff
@@ -198,10 +201,20 @@ public static void addClientCallBody(Function function, String inputCap, Method
             );
             function.addIfElseStatement(reqIsContext.getIfElseStatementNode());
         }
+        if (method.getOutputType() == null && !function.getFunctionDefinitionNode()
+                .functionName().toString().endsWith("Context")) {
+            bindingPatternNode = getTypedBindingPatternNode(
+                    getSimpleNameReferenceNode("var"),
```

Comment:
I think here we can remove `var`, which is relevant to my above comment.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/565/commits/5fae75576479e52517ce4c89b6766f734e739eaf